lib/backdrop.dart

```diff
@@ -136,16 +129,14 @@ class BackdropScaffold extends StatefulWidget {
   /// Creates a backdrop scaffold to be used as a material widget.
   BackdropScaffold({
     this.controller,
-    this.title,
```

Comment:
Can we safely deprecate this somehow maintaining backward compatibility? If `widget.appBar` is not provided then use the old code. Add [Deprecated Annotation](https://api.dart.dev/stable/2.8.2/dart-core/Deprecated-class.html) with proper message to use `BackdropAppBar`. At [`v1.0.0`](https://github.com/fluttercommunity/backdrop/milestone/1) we will deprecate this completely.

Reply:
Ok, I'll have to look up how to do that in dart. But that's a good idea!